### ISSUE TITLE
Relative line numbers and jumping multiple times in a row

### ISSUE THREAD

**Issue by oiiiiiiii** [sxyazi/yazi#1160]
### Please describe the problem you're trying to solve

Coming from ranger, I heavily used the a feature there that allowed me to show the relative distance to the currently selected file:
![grafik](https://github.com/sxyazi/yazi/assets/63345269/7ccd24eb-8f09-401b-b36e-448c4e35f4c2)
In the screenshot example I could then type `3j` to get to yazi.toml. Not as spectacular in this directory, but very helpful in larger ones. This is similar behaviour I use in vim to go to a specific line.

### Would you be willing to contribute this feature?

- [X] Yes, I'll give it a shot

### Describe the solution you'd like

Basically just copy the ranger solution:
1.  Add the option to render relative distance
2. Add the option to repeat commands by typing a number before

### Additional context

I hope it is ok to make this as one feature request, even though it actually contains two. But I feel like they are very related to oneanother and only show there potential in combination.

**Comment by sxyazi** [comment]
Hi, thanks for your interest in Yazi!

I believe https://github.com/dedukun/relative-motions.yazi is what you are looking for. You can find more awesome plugins at https://yazi-rs.github.io/docs/resources

Closing as already exists

**Comment by oiiiiiiii** [comment]
Thank you! I didn't even realize yazi had addons yet. Sorry, should have done more research..
